org.kde.optiimage.json

```diff
@@ -0,0 +1,95 @@
+{
+    "id": "org.kde.optiimage",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.8",
+    "sdk": "org.kde.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "optiimage",
+    "finish-args": [
+        "--filesystem=home",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri"
+    ],
+    "cleanup": ["/bin/aarch64-linux-gnu-gcc"],
+    "modules": [
+        {
+            "name": "jpegoptim",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/tjko/jpegoptim/archive/refs/tags/v1.5.5.tar.gz",
+                    "sha256": "90a309d1c092de358bb411d702281ac3039b489d03adb0bc3c4ef04cf0067d38"
+                }
+            ]
+        },
+        {
+            "name": "oxipng",
+            "buildsystem": "simple",
+            "build-commands": [
+                "ln -s /usr/bin/gcc /app/bin/aarch64-linux-gnu-gcc",
+                "cargo --offline fetch --manifest-path Cargo.toml --verbose",
+                "cargo --offline build --release --verbose",
+                "install -Dm 755 target/release/oxipng -t /app/bin"
+            ],
+            "build-options": {
+                "append-path": "/usr/lib/sdk/rust-stable/bin",
+                "env": {
+                    "CARGO_HOME": "/run/build/oxipng/cargo"
+                }
+            },
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/shssoichiro/oxipng/archive/refs/tags/v9.1.3.tar.gz",
+                    "sha256": "5f34bc3a9eba661a686106261720061b1136301ccd67cc653c9c70d71fa33c09"
+                },
+                "oxipng-sources.json"
+            ]
+        },
+        {
+            "name": "python3-scour",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"scour\" --no-build-isolation"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/75/19/f519ef8aa2f379935a44212c5744e2b3a46173bf04e0110fb7f4af4028c9/scour-0.38.2.tar.gz",
+                    "sha256": "6881ec26660c130c5ecd996ac6f6b03939dd574198f50773f2508b81a68e0daf"
```

Comment:
Same https://release-monitoring.org/project/122409/ (the url is a bit random, if this is hard we can leave it for the future)

Reply:
I added anitya checker data for the others, and for this one I found that there's a [PyPI checker](https://github.com/flathub-infra/flatpak-external-data-checker?tab=readme-ov-file#pypi-checker), so I used that - does that look OK?

If not, I also think we could use the JSON checker.  (I imagine this is what the PyPI checker does internally...)
```json
"x-checker-data": {
    "type": "json",
    "url": "https://pypi.org/pypi/scour/json",
    "version-query": ".info.version",
    "url-query": ".urls[0].url"
}
```

Reply:
that is fine yes